petools/model_tools/pose_classifier/classifier.py

```diff
@@ -0,0 +1,32 @@
+import tensorflow.compat.v1 as tf
+
+from petools.core import ProtobufModel
+from petools.model_tools.transformers.utils import H3P6_2D_NUM
+
+NUM_DIMENSIONS = 3
+
+
+class Classifier(ProtobufModel):
+    """
+    Classify given points as certain pose class
+
+    """
+
+    def __init__(
+            self, protobuf_path: str, session: tf.Session = None):
```

Comment:
Make input dimensionality a parameter of the Classifier. Put it in the constructor.